### DISCUSSION TITLE
[알고리즘] 이진 탐색

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#232] [3. CS(컴퓨터 공학) 관련 글 모음]
요즘 알고리즘 문제 풀이를 시작했는데요, 너무 어렵네요... 기초부터 다시 쌓아야 할 것 같습니다.
[게시글](https://amaran-th.github.io/%EC%95%8C%EA%B3%A0%EB%A6%AC%EC%A6%98/[%EC%95%8C%EA%B3%A0%EB%A6%AC%EC%A6%98]%20%EC%9D%B4%EC%A7%84%20%ED%83%90%EC%83%89/)

**Comment by shkisme** [comment]
으악,,, 아마란스도 자바로 알고리즘 문풀 하기로 했구나
자바로 알고리즘 풀면서 처음으로 자바가 (매우) 짜증났던... ^^;

백준 + 자바로 문풀한다면 인텔리제이에서 라이브 템플릿 등록해두고 사용하는거 추천! [내 블로그에 정리해놓음 !](https://shkisme.vercel.app/IntelliJ-Coding-Snippet)

**Reply by amaran-th** [reply]
헉 천잰데?? 안그래도 일일이 같은 코드 치기 불편했는데ㅋㅋㅋㅋ 땡큐땡큐!!
